Visualization doc: How to scale point size in ParaView (#779)
trixi-framework/TrixiParticles.jl: docs/src/visualization.md

You will now see the following:
![image](https://github.com/user-attachments/assets/383d323a-3020-4232-9dc3-682b0afe8653)

It is useful to make the dot size dependent on the actual particle size.
For this, first make sure you have "fluid_1.pvd" highlighted in the "Pipeline Browser".
Then, in the Properties panel (bottom left), adjust the following settings:
1. "Representation" to "Point Gaussian".
2. Choose the right "Shader Preset": "Plain Circle" for 2D and "Sphere" for 3D.
3. Activate "Scale by Array" and select "`particle_spacing`" in "Gaussian Scale Array".
4. Deactivate "Use Scale Function".
5. Set the "Gaussian Radius" to "`0.5`".
![image](https://github.com/user-attachments/assets/194d9a09-5937-4ee4-b229-07078afe3ff0)

#### Visualization with Macro

> To simplify the visualization of your particle data in ParaView, we've prepared a macro. This macro scales the particles using the `particle_spacing` attribute.
> 

Reviewer: ```suggestion
To simplify the visualization of your particle data in ParaView, you can use a macro.
This macro automates the manual steps in the previous section to a single click of a button.

Install the macro as follows.
```

1. **Save the macro code** (see below) as a `.py` file, e.g. `PointGaussianMacro.py`.
2. Open **ParaView** and go to the top menu:
   **Macros** → **Import New Macro...** → Select the saved `.py` file.
3. The macro will now appear in the **Macros** menu and can optionally be pinned to the **toolbar**.
4. **Load your dataset** into ParaView.
5. **Select the dataset** in the Pipeline Browser.
6. Click on the macro name in the **Macros** menu (or toolbar, if pinned) to run it.
7. The Point Gaussian representation with `particle_spacing` scaling will be applied automatically.

---

> #### Macro Code (Copy and save as `PointGaussianMacro.py`)

Reviewer: ```suggestion
#### Macro Code
```

You will now see the following:
![image](https://github.com/user-attachments/assets/383d323a-3020-4232-9dc3-682b0afe8653)

It is useful to make the dot size dependent on the actual particle size.
For this, **first** make sure you have "fluid_1.pvd" highlighted in the "Pipeline Browser".
Then, in the Properties panel (bottom left), adjust the following settings:
1. "Representation" to "Point Gaussian".
2. Choose the right "Shader Preset": "Plain Circle" for 2D and "Sphere" for 3D.
3. Activate "Scale by Array" and select "`particle_spacing`" in "Gaussian Scale Array".
4. Deactivate "Use Scale Function".
5. Set the "Gaussian Radius" to "`0.5`".

> ![image](https://github.com/user-attachments/assets/6e975d2c-82ed-4d53-936b-bb0beafaf515)

Reviewer: ```suggestion
![image](https://github.com/user-attachments/assets/194d9a09-5937-4ee4-b229-07078afe3ff0)
```
